plugins/mapper-annotated-text/src/main/java/org/elasticsearch/index/mapper/annotatedtext/AnnotatedTextFieldMapper.java

```diff
@@ -58,6 +64,8 @@
  **/
 public class AnnotatedTextFieldMapper extends FieldMapper {
```

Comment:
I think `store` parameter should default to true for this field type if no multi field can be used to synthesize from? (just like text field type)